pkg/trace/info/stats.go

```diff
@@ -255,14 +417,29 @@ func (s *Stats) String() string {
 	eventsExtracted := atomic.LoadInt64(&s.EventsExtracted)
 	eventsSampled := atomic.LoadInt64(&s.EventsSampled)
 
-	return fmt.Sprintf("traces received: %d, traces dropped: %d, traces filtered: %d, "+
+	return fmt.Sprintf("traces received: %d, traces filtered: %d, "+
 		"traces amount: %d bytes, services received: %d, services amount: %d bytes, "+
 		"events extracted: %d, events sampled: %d",
-		tracesReceived, tracesDropped, tracesFiltered,
+		tracesReceived, tracesFiltered,
 		tracesBytes, servicesReceived, servicesBytes,
 		eventsExtracted, eventsSampled)
 }
 
+// WarnString returns a string representation of the Stats struct containing only issues which we should be warning on
+// if there are no issues then an empty string is returned
+func (ts *TagStats) WarnString() string {
```

Comment:
Can we do this in `accStats.Log`?

Reply:
We could, but then we would have to move both of these in there:

* `func (s *Stats) InfoString() string`
* `func (ts *TagStats) WarnString() string`

Which would probably make the `accStats.Log` too cluttered?

Reply:
I don't think so, but I don't feel strongly about it. So if you're convinced that this is better, then leave it. My thought it simply that these aren't used elsewhere. And it's weird why they are called `WarnString` and `InfoString` when the implementation has nothing to do with "warn" and "info". It's more like what they do.

At the very least, it would be nice to lower-case them so that they are not exported as part of the API of this package.